resources/build.js

```diff
@@ -0,0 +1,45 @@
+var execSync = require('child_process').execSync;
+var fs = require('fs');
+var rimraf = require('rimraf');
+var isWindows = /^win/.test(process.platform)
+
+var out = function (command) {
+  execSync(command, {stdio:'inherit'});
+};
+
+console.log('"Running node ./resources/build.js"');
+
+if (!isWindows) {
+  out('set -e'); // exit on console error
```

Comment:
I remember it halting program execution upon an error, but I need to find time to test and prove my memory is either correct or incorrect.